settings.py

```diff
@@ -39,6 +39,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'eventtracking.django',
```

Comment:
As is right now, these settings need to be there because an instance of eventtracking is being mocked. I don't think there is any harm in it.